src/app/containers/MediaPlayer/index.jsx

```diff
@@ -1,55 +1,49 @@
 import React from 'react';
-import Helmet from 'react-helmet';
 import pathOr from 'ramda/src/pathOr';
-import Caption from '../Caption';
-import videoMetadata from './metadata';
-import {
-  NestedGridParentLarge,
-  NestedGridParentSmall,
-  NestedGridItemChildSmall,
-  NestedGridItemChildLarge,
-  GridItemConstrainedLargeNoMargin,
-  GridItemConstrainedSmall,
-} from '../../lib/styledGrid';
-
+import styled from 'styled-components';
+import Amp from './Amp';
+import Canonical from './Canonical';
+import Metadata from './Metadata';
+import embedUrl from './helpers/embedUrl';
+import filterForBlockType from '../../lib/utilities/blockHandlers';
+import useToggle from '../Toggle/useToggle';
+import { RequestContext } from '../../contexts/RequestContext';
+import { GridItemConstrainedMedium } from '../../lib/styledGrid';
 import {
   mediaPlayerPropTypes,
   emptyBlockArrayDefaultProps,
 } from '../../models/propTypes';
-import filterForBlockType from '../../lib/utilities/blockHandlers';
-import { RequestContext } from '../../contexts/RequestContext';
-import useToggle from '../Toggle/useToggle';
 
-const selectWrappers = orientation => {
-  const wrapperSpan = {
-    default: '6',
-    group5: '12',
-  };
-  let ParentWrapper = NestedGridParentLarge;
-  let ChildWrapper = NestedGridItemChildLarge;
-  let Container = GridItemConstrainedLargeNoMargin;
-  let portrait = false;
-
-  if (orientation === 'Portrait') {
-    ParentWrapper = NestedGridParentSmall;
-    ChildWrapper = NestedGridItemChildSmall;
-    Container = GridItemConstrainedSmall;
-    wrapperSpan.default = '4';
-    portrait = true;
-  }
-
-  return { ParentWrapper, ChildWrapper, Container, wrapperSpan, portrait };
+const StyledContainer = styled.div`
+  padding-top: ${({ orientation }) =>
+    orientation === 'Portrait' ? '177.78%' : '56.25%'};
```

Comment:
![](http://giphygifs.s3.amazonaws.com/media/tjwzClJM6fyEw/giphy.gif)

Reply:
<img src="https://media.giphy.com/media/1zhM6uwP4pI6ZB3v0t/giphy.gif" />

It is gross, I agree. Again, something that can be refactored (somehow) in the future?

Reply:
Yeahhhhh, either make them named constants or do the math for it. 

EG: `const landscapeRatio = '56.25%'; // (9/16)*100`

Reply:
I prefer the former to reduce unnecessary (albeit minute) computational power. That or just comment what it's doing

Reply:
Recent commit addresses this, implementing suggestion from @pjlee11